src/model/application/interaction/application_command.rs

```diff
@@ -594,12 +600,15 @@ pub struct CommandDataResolved {
 /// top-level key and another vector of `options`.
 ///
 /// Their resolved objects can be found on [`CommandData::resolved`].
+///
+/// [Discord docs](https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-interaction-data-option-structure).
 #[derive(Clone, Debug, Serialize)]
 #[non_exhaustive]
 pub struct CommandDataOption {
     /// The name of the parameter.
     pub name: String,
     /// The given value.
+    // TODO: Why does this field exist? Why a raw json Value when CommandDataOptionValue exists
```

Comment:
This is about to change in the `next` branch and is already changed for autocomplete data #1941 
```suggestion
```